keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -467,6 +478,12 @@ Allocation will be attempted on the pods in the queue:
 A successful allocation will trigger a pod sync, which will actuate the allocated resize and update the
 pod status accordingly.
 
+### Kubelet-triggered eviction
```

Comment:
Should we add this newly described (to this KEP) eviction behavior to the implementation history so it doesn't seem like we are simultaneously adding new functionality at the same time that we're prepping for GA graduation?